japanese/chasitsu.md

```diff
@@ -2,3 +2,4 @@
 な風景を描いた水墨画と、一輪の水仙を入れた花器が飾られている。
 
 [取り敢えずビールを頼む？](beer/beer.md)
+[庭に出る](garden/garden.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.